js/main2.ts

```diff
@@ -67,27 +61,29 @@ const options = {
 };
 
 export const apple2 = new Apple2(options);
-const cpu = apple2.getCPU();
-const io = apple2.getIO();
+apple2.ready.then(() => {
```

Comment:
This is that race I was talking about in the previous comment. And again, I would stick all this in an `async` function and use `await`.